apps/dashboard/app/(app)/identities/page.tsx

```diff
@@ -1,7 +1,7 @@
 import { OptIn } from "@/components/opt-in";
 import { PageContent } from "@/components/page-content";
 import { Table, TableBody, TableHead, TableHeader, TableRow } from "@/components/ui/table";
-import { getAuth } from "@/lib/auth";
+import { getAuthWithRedirect } from "@/lib/auth";
```

Comment:
I'd vote for "getAuthOrRedirect" to be more precise but it's a nit for sure

Reply:
Fine with me, I had the same thought... After the fact so I had planned to rewrite it again before setting it for review